recipe/meta.yaml

```diff
@@ -24,9 +24,6 @@ outputs:
     script: build_base.sh  # [not win]
     build:
       skip: true  # [py<38]
-      ignore_run_exports:
-        - tk  # [linux]
-        - zlib
     requirements:
       build:
         - pkg-config  # [not win]
```

Comment:
please remove m2-patch from the build: dependencies

Reply:
Done, makes sense, thanks.